### ISSUE TITLE
Alignment issues

### ISSUE THREAD

**Issue by engram-design** [fruitstudios/LinkIt#31]
Has anyone else had the below issues with field alignments? Seems pretty messy? Did just upgrade from 0.9 to 2.3.2 successfully.

<img width="500" alt="screen shot 2016-04-18 at 3 18 26 pm" src="https://cloud.githubusercontent.com/assets/1221575/14594360/00051cf0-0579-11e6-8efc-21c4865f138a.png">

**Comment by samhibberd** [comment]
Yeah, some tidying up to do here, should get around to it when client work calms down :(

**Comment by BillBushee** [comment]
Field alignment is easily manipulated in the plugin's css file (fruitlinkit/resources/css/linkit.css).  This gives me full width text fields stacked vertically.

.fruit-field {
	overflow: hidden;
}
.fruit-field-column {
	/*width: 50%;
	float:left;*/
}
.fruit-field-column:first-child div {
	/*padding-right: 10px;*/
}
.fruit-field-column .checkboxfield {
	padding-top: 5px;
}
.fruitlinkit {}
.fruitlinkit-type {
	/*float:left;
	margin-right: 10px;*/
	margin-bottom:.4rem;
}
.fruitlinkit-type select {
	max-width: 200px;
}
.fruitlinkit-options {
	/*float:left;*/
	margin-bottom:.4rem;
}
.fruitlinkit-settings {
	clear:both;
	margin-bottom:.4rem;
}
